pages/linux/sattach.md

```diff
@@ -0,0 +1,16 @@
+# sattach
+
+> Attach to a Slurm job step.
+> More information: <https://slurm.schedmd.com/sattach.html>.
+
+- Make available the IO streams (`stdout`, `stderr`, and `stdin`) of a Slurm job step:
+
+`sattach {{jobid}}.{{stepid}}`
+
+- Transmit `stdin` to the specified task only:
```

Comment:
```suggestion
- Use the current console's input as `stdin` to the specified task:
```

Not sure how to formulate this better.

Reply:
That suggestion reads much better - let's do that.